sdk/servicebus/service-bus/src/serviceBusMessage.ts

```diff
@@ -572,18 +577,18 @@ export function fromRheaMessage(
       props.lockedUntilUtc = new Date(msg.message_annotations[Constants.lockedUntil] as number);
     }
   }
-  if (msg.ttl != null && msg.ttl >= Constants.maxDurationValue - props.enqueuedTimeUtc.getTime()) {
-    props.expiresAtUtc = new Date(Constants.maxDurationValue);
-  } else {
-    props.expiresAtUtc = new Date(props.enqueuedTimeUtc.getTime() + msg.ttl!);
+  if (msg.ttl == null) msg.ttl = Constants.maxDurationValue;
+  if (props.enqueuedTimeUtc) {
+    props.expiresAtUtc = new Date(
+      Math.min(props.enqueuedTimeUtc.getTime() + msg.ttl, Constants.maxDurationValue)
+    );
   }
 
   const rawMessage = AmqpAnnotatedMessage.fromRheaMessage(msg);
   rawMessage.bodyType = bodyType;
 
   const rcvdsbmsg: ServiceBusReceivedMessage = {
     _rawAmqpMessage: rawMessage,
-    _delivery: delivery,
```

Comment:
Curious why this was removed. Were we just never using it anywhere?

Reply:
I don't see it used anywhere in our code so I'm assuming it's just a relic from a time long past.

Reply:
Removed because '_delivery' does not exist in type 'ServiceBusReceivedMessage'.

It's weird, TS complained about it now only because the type for props is changed from "any".